tests/unit/expected_test_results/parse_origins_from_text/real-list-fr.json

```diff
@@ -0,0 +1,12 @@
+{
+   "lc" : "fr",
+   "origin_fr" : "amandes d’Espagne. Chocolat noir de Côte d’Ivoire. Huile de noisette d’Italie. sucre de France. noisettes d’Italie. fèves de cacao de Côte d’Ivoire. Framboises lyophilisées d’Espagne. arôme naturel de framboise fabriqué en France.",
+   "specific_ingredients" : [
+      {
+         "id" : "en:sugar",
+         "ingredient" : "sucre",
+         "origins" : "en:france",
+         "text" : "sucre de France."
+      }
```

Comment:
This is a bug, isn't it @stephanegigandet ?
We should have a lot more origins in specific_ingredients